lib/docbuild.js

```diff
@@ -0,0 +1,10 @@
+var createDocs = require('node-offline-api').createDocs
+var buildOptions = require('node-offline-api').buildOptions
+var fs = require('fs')
+
+buildOptions.buildDir = process.cwd()
+buildOptions.buildName = 'node_apidoc'
+
+fs.stat('../node-apidoc', function (err) {
```

Comment:
I try to avoid modules without an `export`, can this module also wrap the operation in a module?